metron-analytics/metron-profiler/src/main/java/org/apache/metron/profiler/bolt/ProfileSplitterBolt.java

```diff
@@ -162,15 +156,9 @@ private void doExecute(Tuple input) throws ParseException, UnsupportedEncodingEx
     // ensure there is a valid profiler configuration
     ProfilerConfig config = getProfilerConfig();
     if(config != null && config.getProfiles().size() > 0) {
+      routeMessage(input, message, config);
 
-      // what time is it?
```

Comment:
Here I removed the timestamp logic from Storm.  The `MessageRouter` does this for us now.